sdk/appplatform/azure-mgmt-appplatform/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 10.0.1 (TBD)
+
+### Other Changes
+
+- This package has been deprecated and will no longer be maintained after 03-31-2025. This package will only receive security fixes until 03-31-2025.
```

Comment:
Add retirement declaration link https://aka.ms/asaretirement from service team to changelog.